internal/rbd/rbd_journal.go

```diff
@@ -278,7 +278,7 @@ func (rv *rbdVolume) Exists(ctx context.Context, parentVol *rbdVolume) (bool, er
 
 	// NOTE: Return volsize should be on-disk volsize, not request vol size, so
 	// save it for size checks before fetching image data
-	requestSize := rv.VolSize
+	requestSize := rv.VolSize //nolint:ifshort
```

Comment:
Is this really required? Maybe leave a comment that ifshort fails to detect this (little strange) usage.
The function does a lot more than just checking `Exists()`, so maybe it would be good to rename and split the function in helpers that make this `nolint` annotation unneeded. Can be marked as a `FIXME` in the nolint-comment.